modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -23,6 +23,13 @@ You run a DPDK checkup by performing the following steps:
 * You have installed the OpenShift CLI (`oc`).
 * You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
 
++
+[WARNING]
+====
+The traffic generator pod used by this checkup requires high privileges.
```

Comment:
Can you please clarify what you mean by `high privileges`? Having access to the cluster as a user with cluster-admin permissions is listed as a prerequisite. 

Would it make more sense to just add the specific role requirement as a prerequisite rather than as an admonition?

Reply:
The traffic generator pod has root privileges, and bind mounts to the node's file system.
Combined with the fact that the image is pulled from upstream sources - it is a potential attack surface.